public/js/test/sm-test-case/webpack.config.js

```diff
@@ -0,0 +1,8 @@
+
+module.exports = {
+  entry: "./entry.js",
+  output: {
+    filename: "bundle.js"
+  },
+  devtool: "sourcemap"
+}
```

Comment:
I'd like the debugger test cases to all be hosted so that a user could go to them. Somewhat like mochitests examples as well. 

I think there's a real benefit for contributors to be able to quickly load a test example and see how it works.

> re: never changing

I don't see the examples changing too much, but if they do then I think it is okay to update the tests that are coupled.

One thing to keep in mind is that it's important to be able to run the unit tests in the browser for devtools debugging. So, please try and make sure that the new tests can run with `npm run mocha-server`